Inconsistent build errors and high CPU on `develop`

## Observed behavior
When running `develop`, I have been running into several issues -- including high CPU usage and some errors.

For both of the logs below, CPU started at around 110% right when running `devserver-hot` and consistently crept up, peaking at around 205% right as the build finished. It began to decrease steadily after that, to around 120%. When not using kolibri, but it is still running (as is when I am writing this issue), CPU continues to hover around 120%. As soon as I begin using kolibri again, it averages between 130-150% depending on what tasks I'm completing.

## Errors and logs
[Logs zip](https://github.com/learningequality/kolibri/files/8481544/Archive.zip) -- contains both the non-working and working build (since even the working build seems to have errors at the start that printed in the terminal).

### In the terminal 

#### First time running `devserver-hot` 
results in the 

> This site can’t be reached. localhost refused to connect.

page in the browser. Seems to be a hashi error

```
yarn run devserver-hot
yarn run v1.22.10
$ run-p python-devserver lint-frontend:watch:format hashi-dev watch-hot

> kolibri-root@0.12.0-dev.1 watch-hot
> yarn run watch --hot


> kolibri-root@0.12.0-dev.1 python-devserver
> kolibri start --debug --foreground --port=8000 --settings=kolibri.deployment.default.settings.dev


> kolibri-root@0.12.0-dev.1 lint-frontend:watch:format
> yarn run lint-frontend --monitor --write


> kolibri-root@0.12.0-dev.1 hashi-dev
> yarn workspace hashi run dev

$ kolibri-tools build dev --file ./build_tools/build_plugins.txt --hot
$ kolibri-tools lint --pattern '{kolibri*/**/assets,packages,build_tools}/**/*.{js,vue,scss,less,css}' --ignore '**/dist/**,**/node_modules/**,**/static/**,**/kolibri-core-for-export/**' --monitor --write
npm ERR! invalid options argument
npm ERR! invalid options argument

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/marcellamaki/.npm/_logs/2022-04-13T12_27_54_531Z-debug.log
npm ERR! invalid options argument
npm ERR! invalid options argument

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/marcellamaki/.npm/_logs/2022-04-13T12_27_54_540Z-debug.log

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/marcellamaki/.npm/_logs/2022-04-13T12_27_54_558Z-debug.log

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/marcellamaki/.npm/_logs/2022-04-13T12_27_54_578Z-debug.log
ERROR: "lint-frontend:watch:format" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
(kolibri) bash-3.2$ Kolibri Linter: Initializing watcher for the following patterns: {kolibri*/**/assets,packages,build_tools}/**/*.{js,vue,scss,less,css}
Gathering relevant modules from ['kolibri.core', 'kolibri.plugins.*']
Writing webpack_json output to /var/folders/9g/tky_78fs63j2sly2ddzfbc2h0000gn/T/2022313-90416-2u651n.4w2sl.json
INFO: Writing webpack_json output to /var/folders/9g/tky_78fs63j2sly2ddzfbc2h0000gn/T/2022313-90416-2u651n.4w2sl.json
(node:90416) [DEP_WEBPACK_WATCH_WITHOUT_CALLBACK] DeprecationWarning: A 'callback' argument need to be provided to the 'webpack(options, callback)' function when the 'watch' option is set. There is no way to handle the 'watch' option without a callback.
(Use `node --trace-deprecation ...` to show where the warning was created)
Kolibri Build: Building 'kolibri.core.default_frontend' bundle...
Kolibri Build: Building 'kolibri.core.frontend_head_assets' bundle...
Kolibri Build: Building 'kolibri.plugins.slideshow_viewer.main' bundle...
Kolibri Build: Building 'kolibri.plugins.html5_viewer.main' bundle...
Kolibri Build: Building 'kolibri.plugins.demo_server.main' bundle...
Kolibri Build: Building 'kolibri.plugins.learn.app' bundle...
Kolibri Build: Building 'kolibri.plugins.learn.side_nav' bundle...
Kolibri Build: Building 'kolibri.plugins.perseus_viewer.main' bundle...
Kolibri Build: Building 'kolibri.plugins.pdf_viewer.main' bundle...
Kolibri Build: Building 'kolibri.plugins.user_auth.app' bundle...
Kolibri Build: Building 'kolibri.plugins.user_auth.login_side_nav' bundle...
Kolibri Build: Building 'kolibri.plugins.setup_wizard.app' bundle...
Kolibri Build: Building 'kolibri.plugins.facility.app' bundle...
Kolibri Build: Building 'kolibri.plugins.facility.side_nav' bundle...
Kolibri Build: Building 'kolibri.plugins.coach.app' bundle...
Kolibri Build: Building 'kolibri.plugins.coach.side_nav' bundle...
Kolibri Build: Building 'kolibri.plugins.epub_viewer.main' bundle...
Kolibri Build: Building 'kolibri.plugins.device.app' bundle...
Kolibri Build: Building 'kolibri.plugins.device.side_nav' bundle...
Kolibri Build: Building 'kolibri.plugins.media_player.main' bundle...
Kolibri Build: Building 'kolibri.plugins.user_profile.app' bundle...
Kolibri Build: Building 'kolibri.plugins.user_profile.user_profile_side_nav' bundle...
<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://localhost:3000/
<i> [webpack-dev-server] On Your Network (IPv4): http://192.168.1.228:3000/
<i> [webpack-dev-server] On Your Network (IPv6): http://[fe80::1]:3000/
<i> [webpack-dev-server] Content not from webpack is served from '/Users/marcellamaki/Documents/learning-equality/kolibri/public' directory
Kolibri Build: Completed 'kolibri.core.frontend_head_assets' bundle in 2.792s!
Kolibri Build: Completed 'kolibri.plugins.device.side_nav' bundle in 60.313s!
Kolibri Build: Completed 'kolibri.plugins.user_auth.login_side_nav' bundle in 60.349s!
Kolibri Build: Completed 'kolibri.plugins.learn.side_nav' bundle in 61.008s!
Kolibri Build: Completed 'kolibri.plugins.user_profile.user_profile_side_nav' bundle in 60.984s!
Kolibri Build: Completed 'kolibri.plugins.coach.side_nav' bundle in 61.007s!
Kolibri Build: Completed 'kolibri.plugins.facility.side_nav' bundle in 61.394s!
Kolibri Build: Completed 'kolibri.plugins.demo_server.main' bundle in 62.455s!
Kolibri Build: Completed 'kolibri.plugins.user_profile.app' bundle in 67.522s!
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
Kolibri Build: Completed 'kolibri.plugins.pdf_viewer.main' bundle in 71.914s!
Kolibri Build: Completed 'kolibri.plugins.html5_viewer.main' bundle in 73.322s!
Kolibri Build: Completed 'kolibri.plugins.user_auth.app' bundle in 73.464s!
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
Kolibri Build: Completed 'kolibri.plugins.perseus_viewer.main' bundle in 78.085s!
Kolibri Build: Completed 'kolibri.plugins.slideshow_viewer.main' bundle in 79.025s!
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
Kolibri Build: Completed 'kolibri.plugins.setup_wizard.app' bundle in 80.357s!
Kolibri Build: Completed 'kolibri.plugins.media_player.main' bundle in 82.324s!
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
Kolibri Build: Completed 'kolibri.plugins.device.app' bundle in 83.824s!
Kolibri Build: Completed 'kolibri.plugins.epub_viewer.main' bundle in 85.816s!
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
Kolibri Build: Completed 'kolibri.plugins.coach.app' bundle in 88.282s!
Kolibri Build: Completed 'kolibri.plugins.facility.app' bundle in 88.752s!
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
Kolibri Build: Completed 'kolibri.core.default_frontend' bundle in 93.45s!
Kolibri Build: Completed 'kolibri.plugins.learn.app' bundle in 95.25s!
Kolibri Build: Build complete
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.

```

#### Second time, going to `http://127.0.0.1:8000/` works

```
 yarn run devserver-hot
yarn run v1.22.10
$ run-p python-devserver lint-frontend:watch:format hashi-dev watch-hot

> kolibri-root@0.12.0-dev.1 hashi-dev
> yarn workspace hashi run dev


> kolibri-root@0.12.0-dev.1 lint-frontend:watch:format
> yarn run lint-frontend --monitor --write


> kolibri-root@0.12.0-dev.1 watch-hot
> yarn run watch --hot


> kolibri-root@0.12.0-dev.1 python-devserver
> kolibri start --debug --foreground --port=8000 --settings=kolibri.deployment.default.settings.dev

npm ERR! invalid options argument
npm ERR! invalid options argument
npm ERR! invalid options argument
npm ERR! invalid options argument

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/marcellamaki/.npm/_logs/2022-04-13T12_31_06_663Z-debug.log

npm ERR! A complete log of this run can be found in:
npm ERR!
     /Users/marcellamaki/.npm/_logs/2022-04-13T12_31_06_679Z-debug.log
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/marcellamaki/.npm/_logs/2022-04-13T12_31_06_678Z-debug.log

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/marcellamaki/.npm/_logs/2022-04-13T12_31_06_687Z-debug.log
ERROR: "watch-hot" exited with 1.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
$ kolibri-tools build dev --file ./build_tools/build_plugins.txt --hot
$ kolibri-tools lint --pattern '{kolibri*/**/assets,packages,build_tools}/**/*.{js,vue,scss,less,css}' --ignore '**/dist/**,**/node_modules/**,**/static/**,**/kolibri-core-for-export/**' --monitor --write
$ yarn run build-base --mode=development --watch
INFO     No C extensions are available for this platform
$ webpack --config ./webpack.config.js --mode=development --watch
INFO     Option DEBUG in section [Server] being overridden by environment variable KOLIBRI_DEBUG
INFO     Option DEBUG_LOG_DATABASE in section [Server] being overridden by environment variable KOLIBRI_DEBUG_LOG_DATABASE
INFO     Option RUN_MODE in section [Deployment] being overridden by environment variable KOLIBRI_RUN_MODE
DEBUG    Distro detected as 'Darwin'
DEBUG    Using '<class 'ifcfg.parser.MacOSXParser'>'
DEBUG    Loaded kolibri plugin: kolibri.core
Kolibri Linter: Initializing watcher for the following patterns: {kolibri*/**/assets,packages,build_tools}/**/*.{js,vue,scss,less,css}
DEBUG    Loaded kolibri plugin: kolibri.core.auth
DEBUG    Loaded kolibri plugin: kolibri.core.logger
DEBUG    Loaded kolibri plugin: kolibri.core.notifications
DEBUG    Loaded kolibri plugin: kolibri.core.exams
DEBUG    Loaded kolibri plugin: kolibri.core.device
DEBUG    Loaded kolibri plugin: kolibri.core.lessons
INFO     Running Kolibri with the following settings: kolibri.deployment.default.settings.dev
Gathering relevant modules from ['kolibri.core', 'kolibri.plugins.*']
Writing webpack_json output to /var/folders/9g/tky_78fs63j2sly2ddzfbc2h0000gn/T/2022313-90501-tqax22.ux0nq.json
INFO: Writing webpack_json output to /var/folders/9g/tky_78fs63j2sly2ddzfbc2h0000gn/T/2022313-90501-tqax22.ux0nq.json
INFO     Importing 'tasks' module from django apps
DEBUG    Successfully registered 'kolibri.core.content.tasks.importcontentfromdisk' as job.
DEBUG    Successfully registered 'kolibri.plugins.setup_wizard.tasks.startprovisionsoud' as job.
INFO     Starting Kolibri 0.16.0a4.dev0+git.220.gd22ae65e
DEBUG    MiddlewareNotUsed('kolibri.core.analytics.middleware.MetricsMiddleware'): Request profiling is not enabled
INFO     Bus state: ENTER
INFO     Listening for SIGTERM.
INFO     Listening for SIGHUP.
INFO     Listening for SIGUSR1.
INFO     Listening for SIGINT.
INFO     Bus state: IDLE
INFO     Bus state: START
DEBUG    Successfully registered 'kolibri.core.analytics.utils._ping' as job.
INFO     Starting async task workers.
DEBUG    Started new JOBCHECKER thread ID#298f35f7e2c945b79eb4c1c34b084c4e
DEBUG    Started new SCHEDULECHECKER thread ID#f51e34e5e9254e18a40f3c33de68bb0c
INFO     Started monitor thread 'Autoreloader'.
INFO     Started monitor thread 'ZeroConfPlugin'.
INFO     Started monitor thread 'ProcessControlPlugin'.
INFO     Starting on 0.0.0.0:0
INFO     Waiting for 0.0.0.0:59137
INFO     Serving on 0.0.0.0:59137
INFO     Starting on 0.0.0.0:8000
INFO     Waiting for 0.0.0.0:8000
INFO     Serving on 0.0.0.0:8000
DEBUG    Successfully registered 'kolibri.core.public.utils.soud_sync_cleanup' as job.
INFO     Registering ourselves to zeroconf network with id 'b87bfac924169af7e68aa65691543483' and port '8000'
(node:90501) [DEP_WEBPACK_WATCH_WITHOUT_CALLBACK] DeprecationWarning: A 'callback' argument need to be provided to the 'webpack(options, callback)' function when the 'watch' option is set. There is no way to handle the 'watch' option without a callback.
(Use `node --trace-deprecation ...` to show where the warning was created)
Kolibri Build: Building 'kolibri.core.default_frontend' bundle...
Kolibri Build: Building 'kolibri.core.frontend_head_assets' bundle...
Kolibri Build: Building 'kolibri.plugins.slideshow_viewer.main' bundle...
Kolibri Build: Building 'kolibri.plugins.html5_viewer.main' bundle...
Kolibri Build: Building 'kolibri.plugins.demo_server.main' bundle...
Kolibri Build: Building 'kolibri.plugins.learn.app' bundle...
Kolibri Build: Building 'kolibri.plugins.learn.side_nav' bundle...
Kolibri Build: Building 'kolibri.plugins.perseus_viewer.main' bundle...
Kolibri Build: Building 'kolibri.plugins.pdf_viewer.main' bundle...
Kolibri Build: Building 'kolibri.plugins.user_auth.app' bundle...
Kolibri Build: Building 'kolibri.plugins.user_auth.login_side_nav' bundle...
Kolibri Build: Building 'kolibri.plugins.setup_wizard.app' bundle...
Kolibri Build: Building 'kolibri.plugins.facility.app' bundle...
Kolibri Build: Building 'kolibri.plugins.facility.side_nav' bundle...
Kolibri Build: Building 'kolibri.plugins.coach.app' bundle...
Kolibri Build: Building 'kolibri.plugins.coach.side_nav' bundle...
Kolibri Build: Building 'kolibri.plugins.epub_viewer.main' bundle...
Kolibri Build: Building 'kolibri.plugins.device.app' bundle...
Kolibri Build: Building 'kolibri.plugins.device.side_nav' bundle...
Kolibri Build: Building 'kolibri.plugins.media_player.main' bundle...
Kolibri Build: Building 'kolibri.plugins.user_profile.app' bundle...
Kolibri Build: Building 'kolibri.plugins.user_profile.user_profile_side_nav' bundle...
INFO     Kolibri running on: http://192.168.1.228:8000/
INFO     Kolibri running on: http://127.0.0.1:8000/
INFO     Bus state: RUN
<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://localhost:3000/
<i> [webpack-dev-server] On Your Network (IPv4): http://192.168.1.228:3000/
<i> [webpack-dev-server] On Your Network (IPv6): http://[fe80::1]:3000/
<i> [webpack-dev-server] Content not from webpack is served from '/Users/marcellamaki/Documents/learning-equality/kolibri/public' directory
assets by status 948 KiB [cached] 4 assets
runtime modules 8.93 KiB 14 modules
orphan modules 15.9 KiB [orphan] 10 modules
javascript modules 668 KiB
  modules by path ./ 372 KiB
    modules by path ./node_modules/core-js/ 172 KiB 157 modules
    modules by path ./src/ 197 KiB 19 modules
    modules by path ./node_modules/uuid/dist/esm-browser/*.js 3.24 KiB 5 modules
  modules by path ../../node_modules/ 296 KiB
    modules by path ../../node_modules/lodash/*.js 126 KiB 163 modules
    modules by path ../../node_modules/toposort-class/ 7.96 KiB 2 modules
json modules 1.12 KiB
  ./h5p_build.json 66 bytes [built] [code generated]
  ./src/mimetypes.json 1.05 KiB [built] [code generated]
webpack 5.32.0 compiled successfully in 5289 ms
Kolibri Build: Completed 'kolibri.core.frontend_head_assets' bundle in 3.032s!
Kolibri Build: Completed 'kolibri.plugins.device.side_nav' bundle in 78.32s!
Kolibri Build: Completed 'kolibri.plugins.user_auth.login_side_nav' bundle in 78.455s!
Kolibri Build: Completed 'kolibri.plugins.user_profile.user_profile_side_nav' bundle in 81.03s!
Kolibri Build: Completed 'kolibri.plugins.coach.side_nav' bundle in 81.055s!
Kolibri Build: Completed 'kolibri.plugins.learn.side_nav' bundle in 81.114s!
Kolibri Build: Completed 'kolibri.plugins.facility.side_nav' bundle in 81.436s!
Kolibri Build: Completed 'kolibri.plugins.demo_server.main' bundle in 83.289s!
Kolibri Build: Completed 'kolibri.plugins.user_profile.app' bundle in 88.459s!
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
Kolibri Build: Completed 'kolibri.plugins.pdf_viewer.main' bundle in 106.503s!
Kolibri Build: Completed 'kolibri.plugins.user_auth.app' bundle in 107.112s!
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
Kolibri Build: Completed 'kolibri.plugins.html5_viewer.main' bundle in 108.23s!
Kolibri Build: Completed 'kolibri.plugins.perseus_viewer.main' bundle in 109.558s!
Kolibri Build: Completed 'kolibri.plugins.slideshow_viewer.main' bundle in 110.756s!
Kolibri Build: Completed 'kolibri.plugins.setup_wizard.app' bundle in 111.094s!
Kolibri Build: Completed 'kolibri.plugins.media_player.main' bundle in 111.26s!
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
Kolibri Build: Completed 'kolibri.plugins.device.app' bundle in 112.896s!
Kolibri Build: Completed 'kolibri.plugins.epub_viewer.main' bundle in 113.506s!
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
Kolibri Build: Completed 'kolibri.plugins.facility.app' bundle in 114.662s!
Kolibri Build: Completed 'kolibri.plugins.coach.app' bundle in 118.185s!
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
Kolibri Build: Completed 'kolibri.core.default_frontend' bundle in 125.145s!
Kolibri Build: Completed 'kolibri.plugins.learn.app' bundle in 126.964s!
Kolibri Build: Build complete
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
<i> [webpack.cache.PackFileCacheStrategy/webpack.FileSystemInfo] Node.js doesn't offer a (nice) way to introspect the ESM dependency graph yet.
<i> Until a full solution is available webpack uses an experimental ESM tracking based on parsing.
<i> As best effort webpack parses the ESM files to guess dependencies. But this can lead to expensive and incorrect tracking.
```

## Steps to reproduce
The error happens frequently while trying to reproduce or work on develop, and it is consistently, but unpredictably reproducible. I haven't been able to figure out _why_ the errors happen sometimes and not others, although there do seem to be _different_ errors in the build that indicate whether or not it's going to actually run.


## Context
Kolibri develop
Mac OS Big Sur 11.6.2
Not a mac with an M1 chip
running on chrome

--- a/packages/kolibri-tools/lib/webpack.config.plugin.js
+++ b/packages/kolibri-tools/lib/webpack.config.plugin.js
@@ -172,7 +172,6 @@ module.exports = (
     bundle.watch = true;
     bundle.watchOptions = {
       aggregateTimeout: 300,
-      poll: 1000,
     };
   }
 

